phpBB/language/en/acp/posting.php

```diff
@@ -53,6 +53,9 @@
 	'BBCODE_HELPLINE_EXPLAIN'	=> 'This field contains the mouse over text of the BBCode.',
 	'BBCODE_HELPLINE_TEXT'		=> 'Help line text',
 	'BBCODE_HELPLINE_TOO_LONG'	=> 'The help line you entered is too long.',
+	'BBCODE_FONT_ICON'			=> 'BBCode icon',
+	'BBCODE_FONT_ICON_EXPLAIN'	=> 'Enter the name of a Font Awesome icon (without the fa prefix) to display instead of the BBCode name appearing on the button. <a href="https://fontawesome.com/v6/icons/" target="_blank">Click here</a> to view the list of available icons.',
```

Comment:
HTML should be defined in the template file and ideally you should only link to the free icons. Also, maybe mention that only solid icons are supported?

Reply:
Sure thing, I'll tweak this.